doc/prompt_task_map.json

```diff
@@ -0,0 +1,746 @@
+{
+    "commonsense_qa": [
+        {
+            "task": "task073_commonsenseqa_answer_generation",
+            "prompt": [
+                "most_suitable_answer",
+				"question_answering",
+				"question_to_answer_index"
+            ]
+        }
+    ],
+    "dream": [
+        {
+            "task": "task247_dream_answer_generation",
+            "prompt": [
+                "baseline",
+				"read_the_following_conversation_and_answer_the_question"
+            ]
+        }
+    ],
+    "quail": [
+        {
+            "task": "task887_quail_answer_generation",
+            "prompt": [
+                "context_description_question_text",
+				"context_question_description_text",
+				"description_context_question_text"
+            ]
+        }
+    ],
+    "quartz": [
+    ],
+    "social_i_qa": [
+        {
+            "task": "task384_socialiqa_question_classification",
+            "prompt": [
+				"Check if a random answer is valid or not"
+            ]
+        },
+        {
+            "task": "task580_socialiqa_answer_generation",
+            "prompt": [
+				"Show choices and generate answer",
+				"Show choices and generate index"
+            ]
+        }
+    ],
+    "wiqa": [
+    ],
+    "cosmos_qa": [
+        {
+            "task": "task024_cosmosqa_answer_generation",
+            "prompt": [
+				"context_description_question_text",
+				"context_question_description_text",
+				"description_context_question_text"
+            ]
+        }
+    ],
+    "qasc": [
```

Comment:
We have qasc tasks right? `task040_qasc_question_generation`, `task041_qasc_answer_generation`.

Reply:
Yes, but we don't have any shared task with them.